### ISSUE TITLE
Copy-Paste of Outputwindow is not possible with Ctrl + C

### ISSUE THREAD

**Issue by kallitokaco** [cnr-isti-vclab/meshlab#176]
On Windows 8.1 it's not possible to copy text with Ctrl + C from the output view.
But its poissible to use right-click copy action to succesfully copy the selected text to the clipboard.
Ctrl + C results in the following clipboard content: 
```
<!DOCTYPE ViewState>
<project>
 <VCGCamera PixelSizeMm="0.0369161 0.0369161" LensDistortion="0 0" CameraType="0" TranslationVector="-2.45326 -13.0524 -55.1799 1" ViewportPx="1585 930" FocalMm="29.7323" CenterPx="792 465" RotationMatrix="-0.999031 0.0412155 0.0154321 0 -0.0366814 -0.973556 0.225485 0 0.0243175 0.224701 0.974124 0 0 0 0 1 "/>
 <ViewSettings FarPlane="5.20735" NearPlane="0.909327" TrackScale="0.0332071"/>
</project>

```

**Comment by mcallieri** [comment]
yes, I noticed it too. the "COPY" action for copying the camera should be active only whern the 3D window has focus, and not when mouse is  on the log. Iìll try fixing it.

**Comment by mcallieri** [comment]
I should have corrected it in f85efd35214e4a63302b79859ec1801f3db0b6bb
Still need some testing
